Newrelic daemonset

azell opened the pull request:
1. base64 on Mac does not support the **wrap** option. Easy to support on both Mac and Linux by using **tr** to remove the newline.
2. DaemonSet definition does not conform to latest schema:

> $ kubectl create -f ./newrelic-daemonset.yaml 
> error validating "./newrelic-daemonset.yaml": error validating data: found invalid field privileged for v1.PodSecurityContext; if you choose to ignore these errors, turn validation off with --validate=false

<!-- Reviewable:start -->

---

This change is [<img src="http://reviewable.k8s.io/review_button.svg" height="35" align="absmiddle" alt="Reviewable"/>](http://reviewable.k8s.io/reviews/kubernetes/kubernetes/24564)

<!-- Reviewable:end -->

thockin commented:
Travis says 

```
/home/travis/gopath/src/github.com/kubernetes/kubernetes/examples/ is out of date. Please run hack/update-generated-docs.sh
!!! Error in hack/../hack/verify-generated-docs.sh:28
  '"${KUBE_ROOT}/hack/after-build/verify-generated-docs.sh" "$@"' exited with status 1
```

piosz commented:
`/home/travis/gopath/src/github.com/kubernetes/kubernetes/examples/ is out of date. Please run hack/update-generated-docs.sh`

wojtek-t commented:
@azell - can you please fix the error @piosz mentioned above?

wojtek-t commented:
@azell - can you please squash the commits?

piosz commented:
Will add lgtm label once the commits are squashed.

azell committed:
The Mac version of base64 does not support the wrap option.  Use tr to delete the trailing newline for Mac and Linux runs.

Move the security context tag under containers.

Run hack/update-generated-docs.sh

wojtek-t commented:
LGTM - thanks!
